From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: thororen <78185467+thororen1234@users.noreply.github.com>
Date: Fri, 16 Oct 2026 00:00:00 -0400
Subject: build: restore macOS 10.13 deployment target

Align Node native code with Electron's legacy macOS deployment target.

diff --git a/common.gypi b/common.gypi
index fd3895604ae198fa4d0fc06a433eeba329b97494..b6e7bb2e2574efbcb0fae80b4874ba7c74bbf5f9 100644
--- a/common.gypi
+++ b/common.gypi
@@ -746,7 +746,7 @@
           'GCC_ENABLE_PASCAL_STRINGS': 'NO',        # No -mpascal-strings
           'GCC_STRICT_ALIASING': 'NO',              # -fno-strict-aliasing
           'PREBINDING': 'NO',                       # No -Wl,-prebind
-          'MACOSX_DEPLOYMENT_TARGET': '13.5',       # -mmacosx-version-min=13.5
+          'MACOSX_DEPLOYMENT_TARGET': '10.13',      # -mmacosx-version-min=10.13
           'USE_HEADERMAP': 'NO',
           'WARNING_CFLAGS': [
             '-Wall',
@@ -774,6 +774,7 @@
           ['target_arch=="arm64"', {
             'xcode_settings': {
               'ARCHS': ['arm64'],
+               'MACOSX_DEPLOYMENT_TARGET': '11.0',
               'OTHER_LDFLAGS!': [
                 '-Wl,-no_pie',
               ],
